Source/WebInspectorUI/UserInterface/Views/LayoutDetailsSidebarPanel.js

```diff
@@ -149,24 +155,30 @@ WI.LayoutDetailsSidebarPanel = class LayoutDetailsSidebarPanel extends WI.DOMDet
 
     // Private
 
-    _handleLayoutContextTypeChanged(event)
+    _handleNodeInserted(event)
     {
-        let domNode = event.target;
+        this._refreshNodeSets();
```

Comment:
This is gonna be extremely hot/busy :/

We probably need to instead need to do this inside `layout` so that we don't try to do this every single time a new node is added to the DOM.  You could probably instead have `this._flexNodeSet = null;` and use that as a signal to refresh it inside `layout`.

NIT: it also might be more efficient to grab _all_ nodes with a `layoutContextType` and then filter them here ourselves instead of walking the entire DOM twice (i.e. get rid of `WI.DOMManager.prototype.nodesWithLayoutContextType` and instead just `Array.from(WI.domManager.attachedNodes({filter: (node) => node.layoutContextType}))`)